lib/cluster/cluster.c

```diff
@@ -26,21 +26,27 @@
 
 CRM_TRACE_INIT_DATA(cluster);
 
+/*!
+ * \brief Get (and set if needed) a node's UUID
+ *
+ * \param[in] peer  Node to check
+ *
+ * \return Node UUID of \p peer, or NULL if unknown
+ */
 const char *
 crm_peer_uuid(crm_node_t *peer)
 {
     char *uuid = NULL;
-    enum cluster_type_e type = get_cluster_type();
 
     // Check simple cases first, to avoid any calls that might block
-    if(peer == NULL) {
+    if (peer == NULL) {
```

Comment:
Would it be worth coming up with an indent file and running all the source through that?

Reply:
There's already the p-indent file, but that hasn't been touched since 2011 so maybe it would be worth starting over.

Reply:
p-indent is probably pretty close to current guidelines, I bet we could update it without too much pain. I'm not keen to see how the code base stacks up though, and filter out false positives, so I've never tried. One thing I hate though is cluttering the code with indent, coverity, cppcheck, etc. overrides. I'd be happier if we could do it without much of that.

Reply:
> p-indent is probably pretty close to current guidelines, I bet we could update it without too much pain. I'm not keen to see how the code base stacks up though, and filter out false positives, so I've never tried.

I was bored, so I tried:
```
clumens@localhost:~/src/pacemaker$ find . -name '*.[ch]' | xargs ./p-indent
clumens@localhost:~/src/pacemaker$ git diff --stat | tail -1
 293 files changed, 17422 insertions(+), 19161 deletions(-)
```

> One thing I hate though is cluttering the code with indent, coverity, cppcheck, etc. overrides. I'd be happier if we could do it without much of that.

I'm sure the maintainer of each tool would tell you to not have all those overrides and just accept the results.

Reply:
More annoyingly, you have to give indent an argument like `-T xmlNode` or it'll turn `void foo(xmlNode *x)` into `void foo(xmlNode * x)`, and you have to do that for every typedef.